src/view/styles.ts

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+export const PageWrapper = styled.div`
+  height: 90vh;
+  margin-top: -2rem;
+  display: flex;
+  justify-content: center;
+`;
+
+export const ContentWrapper = styled.div`
+  width: 40rem;
```

Comment:
[suggestion] Since we're removing the flex display on the container we need to add a `margin: auto` here to center the component.